src/controllers/network.js

```diff
@@ -108,6 +108,34 @@ class Network {
                 return;
             }
 
+            if (_.get(data, 'Driver') === 'overlay') {
+                Log.warn('Detected daemon configuation using OVERLAY NETWORK for server containers.');
+                Log.info('Removing interface address and enabling ispn.');
```

Comment:
I think this should be the other way round. If it detects the networking driver thats fine. But when it changes the config that should result in a warning.